### ISSUE TITLE
My trained model only generates noises

### ISSUE THREAD

**Issue by jstyoon96** [beresandras/clear-diffusion-keras#4]
nice work of your diffusion model, however training results were generating only noises instead of images. what possibly could be wrong?
v_loss, i_loss, n_loss doesn't seem to get lower.
parameters were as set to default as mentioned and also tried changing the prediction_type to "velocity"
below are the loss chages after 80 epochs
![image](https://user-images.githubusercontent.com/82366908/202965212-486f89b9-d89d-4875-8561-f31381186501.png)

**Comment by beresandras** [comment]
Hi!

After 3-4 epochs you should already see some reasonable images. Did you see loss spikes during training? Did you change anything (network, dataset, resolution, hyperparams)? The loss usually stops decreasing after training for lots of epochs, but it should be already much lower, n_loss should be well below 0.2.
